clevertap-core/src/main/java/com/clevertap/android/sdk/inbox/CTInboxListViewFragment.java

```diff
@@ -155,12 +159,24 @@ public void run() {
             recyclerView.setLayoutManager(linearLayoutManager);
             recyclerView.addItemDecoration(new VerticalSpaceItemDecoration(18));
             recyclerView.setItemAnimator(new DefaultItemAnimator());
+            applySystemBarsInsets(recyclerView);
             recyclerView.setAdapter(inboxMessageAdapter);
             inboxMessageAdapter.notifyDataSetChanged();
         }
         return allView;
     }
 
+    private void applySystemBarsInsets(RecyclerView view) {
+        view.setClipToPadding(false);
+        ViewCompat.setOnApplyWindowInsetsListener(view, (v, insets) -> {
+            Insets bars = insets.getInsets(
+                    WindowInsetsCompat.Type.systemBars()
+            );
+            v.setPadding(bars.left, 0, bars.right, bars.bottom);
```

Comment:
we are using paddings where instead of margins, also why do we not do this on parent and do it on RV?

Reply:
Resolving as discussed.